README.md

```diff
@@ -35,7 +35,7 @@ description: example
 
 * `version` is a **required** parameter that indicates the configure file's version, the default version is `v2rc1`.
 * `description` is an **optional** parameter that describes the configure file.
-* `clientSettings` takes care all the Nebula related configurations.
```

Comment:
`clientSettings` takes care all the Nebula related configurations.

All configurations related to the **Nebula Graph** server are configured in the`clientSetting` field.